### DISCUSSION TITLE
Can I preview html/markdown files in text, not a webpage?

### DISCUSSION THREAD

**Discussion by MC-dusk** [QL-Win/QuickLook#1152] [Q&A] [answered]
I'm wondering how to just preview html files in text. Now it will show me webpage directly.

And the same problem for previewing markdown files.

Because it's slow and most of time I just want to have a look on the code, not the display effects.

Thank you!

**Comment by MC-dusk** [comment]
Can I just modify settings on my PC?

**Comment by rabelux** [comment]
Unfortunately this hasn't been implemented yet and I don't think the dev will find time to do it. PRs are always welcome.
Related issue: #336

**Reply by MC-dusk** [reply, marked as answer]
That's confusing. The requirement is not a new function, it can just use the existing view mode.

You can just view md file as txt file.

It's just like deleting some extra code to recover the original function, I think.

Any way, thanks.
